package.json

```diff
@@ -26,14 +26,14 @@
   },
   "homepage": "https://github.com/webpack-contrib/voting-app#readme",
   "devDependencies": {
+    "@babel/core": "^7.7.4",
+    "@babel/preset-env": "^7.7.4",
+    "@babel/preset-react": "^7.7.4",
+    "@babel/register": "^7.9.0",
```

Comment:
`"@babel/register"` was required as I think they moved out the register from the core